include/thingset/can.h

```diff
@@ -193,6 +193,14 @@ extern "C" {
      && THINGSET_CAN_PRIO_GET(id) >= 4)
 #define THINGSET_CAN_REQRESP(id) ((id & THINGSET_CAN_TYPE_MASK) == THINGSET_CAN_TYPE_REQRESP)
 
+/**
+ * Callback typedef for received single-frame reports (type 0x2) via CAN
```

Comment:
I'm not sure this description is correct.

Reply:
Thanks, fixed.